olp-cpp-sdk-core/src/cache/DefaultCacheImpl.cpp

```diff
@@ -93,6 +95,11 @@ leveldb::CompressionType GetCompression(
              : leveldb::kSnappyCompression;
 }
 
+int64_t GetElapsedTimeFromStart(std::chrono::steady_clock::time_point start) {
```

Comment:
```suggestion
int64_t GetElapsedTime(std::chrono::steady_clock::time_point start) {
```
It is quite obvious that it is from start, no need to extra mention it.